### DISCUSSION TITLE
some Error when i run the crazyswarm.launch

### DISCUSSION THREAD

**Discussion by LESLIEWANGer** [USC-ACTLab/crazyswarm#492] [Q&A]
hello @whoenig 
I installed the noetic and crazyswarm on ubuntu 20.04 as guidance,also I can receive the information about cf's battery and  position via vrpn.
But when i run crazyswarm.launch.nothing happens,I followed the NatNetSDKCrosspatform,it occuerd.
![2021-10-03 11-58-16 的屏幕截图](https://user-images.githubusercontent.com/51128132/135739302-429c2153-22e8-4c86-bee4-b4893555e170.png)
the last line means "NO that file or dictionary".
I think the crazyswarm doesn't receive the position information.
Does it mean I should install natnet on my ubuntu or update the motive on windows(its version is 1.9)?
Thanks!

**Comment by whoenig** [comment]
mcr doesn't exist anymore in the latest version (I need to update the readme). The only relevant part is the output of natnettest.

Does the mocap_helper.launch file work for you?

**Reply by LESLIEWANGer** [reply]
emmm
it also doesn't work .
The rviz appears after running mocap_helper.launch but no plane on it.
It seems that the Motive needs to be updated ?

**Reply by whoenig** [reply]
mocap_helper.launch only prints data on the screen and doesn't start rviz.
This seems to be a bug (see the other discussions). You can either try to fix it and send a pull request or upgrade Motive. It's difficult for me to fix without access to an old Motive version.
